docs/components/ComponentLayout.css

```diff
@@ -125,6 +125,15 @@ ul.Component__list li {
   text-align: center;
 }
 
+table.Component__Props tr td:first-child,
+ul.Component__Props
+  li
+  dl
+  .DescriptionList__item:first-child
+  .DescriptionList__details {
```

Comment:
This is here to apply the same properties to both values equilaterally if we wanted to apply styles to the prop name specifically in the future.